documentation/docs/libraries/rust/cheat_sheet.md

```diff
@@ -398,6 +399,6 @@ account.sign(did, "key-1", &mut credential).await?;
 
 ## Verifiable Presentations (VP)
 
-A Verifiable Presentation is the format in which you can share a (collection of) Verifiable Credential(s). It is signed by the subject, to prove control over the Verifiable Credential with a nonce or timestamp.
+A Verifiable Presentation is the format in which you can share a (collection of) Verifiable Credential(s). It is signed by the subject to prove control over the Verifiable Credential with a nonce or timestamp.
 
-### [Create](../../verifiable_credentials/verifiable_presentations.mdx)
+### [Create](https://wiki.iota.org/identity.rs/verifiable_credentials/verifiable_presentations)
```

Comment:
Relative links allows a locally run version to link to the local resources instead of going to the main deployment. It also is future proof again changes in the top domain name. Why would we go with absolute links?